packages/client/components/ThreadedItemHeaderDescription.tsx

```diff
@@ -19,7 +19,9 @@ const HeaderDescription = styled('div')({
 
 const HeaderName = styled('div')({
   color: PALETTE.SLATE_700,
-  fontWeight: 600
+  fontWeight: 600,
+  overflowWrap: 'break-word',
+  width: 200
```

Comment:
We now have markup broken in the discussion section:

Now:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/466991/214025017-d4ea7c25-155f-455e-9ef7-d7e2dd251b9f.png">

Was:

<img width="413" alt="image" src="https://user-images.githubusercontent.com/466991/214025078-8ed37eea-3da9-4227-ad7f-809c8cbba43d.png">